crates/bevy_render/src/render_asset.rs

```diff
@@ -235,7 +238,10 @@ pub(crate) fn extract_render_asset<A: RenderAsset>(
             let mut removed = <HashSet<_>>::default();
 
             for event in events.read() {
-                #[allow(clippy::match_same_arms)]
+                #[expect(
+                    clippy::match_same_arms,
+                    reason = "LoadedWithDependencies is marked as a TODO, so it's likely this will no longer lint soon."
+                )]
```

Comment:
I am not a fan of this reason... but I also didn't know what else to do, because merging the two arms would mean removing the `// TODO` comment.